src/Altinn.Platform/Altinn.Platform.Storage/Storage.Interface/Models/Hooks.cs

```diff
@@ -0,0 +1,15 @@
+using Newtonsoft.Json;
+namespace Altinn.Platform.Storage.Models
+{
+    /// <summary>
+    /// Model of application hooks
+    /// </summary>
+    public class Hooks
```

Comment:
Drop the Hooks class it doesn't do anything other than imply that there are more hooks.